src/Uno.UI.Runtime.Skia.Gtk/GTK/UnoDrawingArea.cs

```diff
@@ -8,35 +11,55 @@ internal class UnoDrawingArea : Gtk.DrawingArea
 	{
 		private SKBitmap bitmap;
 		private int renderCount;
-		private int InvalidateRenderCount;
 
 		public UnoDrawingArea()
 		{
 			WUX.Window.Current.InvalidateRender
 				+= () =>
 				{
-					QueueDrawArea(0, 0, 10000, 1000);
+					Invalidate();
 				};
 		}
 
+		private void Invalidate()
+			=> QueueDrawArea(0, 0, 10000, 10000);
+
+		private void Screen_MonitorsChanged(object sender, EventArgs e)
+		{
+			UpdateDpi();
+			Invalidate();
+		}
+
+		private void UpdateDpi()
+		{
+		}
+
 		protected override bool OnDrawn(Cairo.Context cr)
 		{
 			int width, height;
 
-			Console.WriteLine($"Render {renderCount++}");
+			if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Trace))
+			{
+				this.Log().Trace($"Render {renderCount++}");
+			}
+
+			var dpi = 1; // (Window.Screen?.Resolution ?? 1) / 96.0;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)